scripts/backend_tests.py

```diff
@@ -215,6 +220,12 @@ def _convert_to_test_target(path):
             result.append(_convert_to_test_target(
                 os.path.join(base_path, root)))
         for subroot, _, files in os.walk(os.path.join(base_path, root)):
+            if subroot.endswith(_LOAD_TESTS_DIR) and include_load_tests:
```

Comment:
Also, if you look at line 217, there's an exclusion for "core/tests". Does that need to be modified?

Basically, does something like `bash scripts/run_backend_tests.sh --test_target=core.tests.load_tests` (or having the test target be a specific file in that directory) work?